### DISCUSSION TITLE
6주차 회고

### DISCUSSION THREAD

**Discussion by JiinSeok** [team-final/albaform#223] [General]
프로젝트 마지막 주에 대해 남겨주세요!
기한: 10/13 일요일

**Comment by JiinSeok** [comment]
# KEEP

**Reply by JiinSeok** [reply]
코드 리뷰가 마지막까지 꾸준하게 이루어졌다.

**Reply by engdawn** [reply]
코드 리뷰가 활발했다.

**Comment by JiinSeok** [comment]
# PROBLEM

**Reply by JiinSeok** [reply]
프로젝트에서 공들인 부분, 뿌듯한 부분에 대해서 라벨, 링크 같은 인덱스를 당시에 두지 않아서 끝나고 문서화할 때 부담된다.

**Reply by engdawn** [reply]
문제 해결에 대한 고민과 관심이 부족했다.

**Comment by JiinSeok** [comment]
# TRY

**Reply by JiinSeok** [reply]
리팩토링, 문서화, 추가개발을 다시 시작할 수 있도록 재시작 회의를 잡고 쉬면 좋을 것 같다

**Reply by JiinSeok** [reply]
매일 개발일지를 쓰면서 개발하면 좋을 것 같다

**Reply by engdawn** [reply]
코드나 프로젝트 과정 중 드는 생각을 짧은 메모로 정리하는 것을 시작하는 것도 좋을 것 같다.
